### PR TITLE
post: monorepo init [closes #242]

### LINKED ISSUE #242: [post] gomonorepo intro
https://gomonorepo.org/

### Inspire or Source

- https://www.tomasvotruba.cz/blog/2017/06/05/go-php-71/
- https://github.com/shopsys/monorepo-tools

### PR DESCRIPTION
**Pull request by TomasVotruba**
### Preview

preview

### CODE REVIEW

File: source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+id: 82
+title: "Monorepo on the Rise in PHP"
+perex: '''
+    Do you know what monorepo is? How to start with it? What you need and do even other programming languages use it? And what about Facebook and Google think about it?
```
Comment (carusogabriel) [inline source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md]:
```diff
-Do you know what monorepo is?
+Do you know what a monorepo is?
```

File: source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Javasript, Java, C++, Android, iOS, Facebook, Google? Monorepo is Standard
+
+When you pop out of PHP bubble, you can see in [awesome-monorepo](https://github.com/korfuri/awesome-monorepo) a collection of monorepo tools, that are used out in a wild by another languages. Google has [Bazel](https://bazel.build/), Facebook has [Buck](https://buckbuild.com/), Twitter and Foursqaure have [Pants](https://www.pantsbuild.org/).
+
+**Still a PHP-related tool is missing**. Why? It might be the case, that no global company that goes open-source uses PHP as their main language.   
+
+## Symfony Monorepo Spreads the Word
+ 
+PHP is aware of monorepo term mainly thanks to [Symfony](https://github.com/symfony/symfony),  that uses it to maintain **all its components, that are split to standalone** read-only repositories, e.g. [Symfony\Console](https://github.com/symfony/console), [Symfony\EventDispatcher](https://github.com/symfony/event-dispatcher).
```
Comment (carusogabriel) [inline source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md]: Double spaces:
```diff
-),  that uses it to
+), that uses it to
```

File: source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+### Tool to Split With
+
+Since Symfony needed a fast tool to split over 30 repositories across 3 and more branches, Fabien Potencier came with [splits](https://twitter.com/fabpot/status/739860138564149248?lang=en) tool.
+
+He had also [very interesting talk about it](https://www.youtube.com/watch?v=4w3-f6Xhvu8) ([slides](https://speakerdeck.com/fabpot/a-monorepo-vs-manyrepos)), where he explains in details, how Symfony monorepo works, what is spliting, what are the needs for a build tool managing monorepo.
```
Comment (carusogabriel) [inline source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md]:
```diff
-He had also [very interesting talk about it]
+He also has a [very interesting talk about it]
```
```diff
-what is spliting, what are the 
+what is splitting and what are the 
```

File: source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+I work on [Shopsys Framework](https://shopsys-framework.com), an open-source e-commerce platform on Symfony and the monorepo topic finally came to the sprint to be realized. If you've read until now, **you know there is not much shared know-how or support for new-commers to adopt a monorepo pattern in PHP**. 
+
+That's how [`shopsys/monorepo-tools`](https://github.com/shopsys/monorepo-tools), that covers both build and split was given birth by [Petr Heinz](https://github.com/PetrHeinz).
+
+It's slowly getting better with monorepo in PHP. But still many people don't know what is difference between *a monorepo* and *a monolith*.
+
+## New to monorepo? Start on GoMonorepo.org
+
+Inspired by [GoPHP71.org](http://gophp71.org/), **to boost this progress even further and answer some basic questions about monorepo, I build a simple site [Gomonorepo.org](https://gomonorepo.org/)**.
```
Comment (carusogabriel) [inline source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md]:
```diff
-http://gophp71.org
+https://gophp71.org
```

@TomasVotruba Gonna create a `.sh` or `php` script to automate this. We love automation, don't we?

File: source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+<img src="/assets/images/posts/2018/monorepo/monorepo-website.png" class="img-thumbnail">
+
+### What you'll Find There?
```
Comment (carusogabriel) [inline source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md]:
```diff
-What you'll Find There?
+What will you Find There?
```

File: source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md (hunk 4 above)
Comment (TomasVotruba) [reply source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md]: I had exactly the same in mind. But how to check the website is `https` ready?

Comment (carusogabriel) [reply source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md]: Maybe ping it and see if a valid response was returned. Gonna see how to do it

Comment (TomasVotruba) [reply source/_posts/2018/2018-03-08-monorepo-on-the-rise-in-php.md]: There is this package I use for existing urls: https://github.com/TomasVotruba/tomasvotruba.cz/blob/master/composer.json#L16

Use like this: https://github.com/TomasVotruba/tomasvotruba.cz/blob/master/composer.json#L38

**Pull request merged**